x-pack/test/security_solution_cypress/cypress/e2e/explore/cases/attach_alert_to_case.cy.ts

```diff
@@ -25,8 +25,7 @@ const loadDetectionsPage = (role: SecurityRoleName) => {
   waitForAlertsToPopulate();
 };
 
-// Failing: See https://github.com/elastic/kibana/issues/191819
-describe.skip('Alerts timeline', { tags: ['@ess'] }, () => {
+describe('Alerts timeline', { tags: ['@ess'] }, () => {
```

Comment:
Now that we support roles on serverless, it would be great on a follow-up PR to enable this test there as well.